### PR TITLE
Timestamp added to text body

### PR DESCRIPTION
**Pull request by txu89**
screen shot 2018-02-08 at 3 16 37 pm
Text should look like above + nvidia URL

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -71,6 +72,7 @@ async function check1070() {
       : '1070 IS IN STOCK!!! 1070 IS IN STOCK!!! 1070 IS IN STOCK!!!';
 
     console.log(`${new Date().toTimeString()}: ${output}`);
+    console.log(moment().format('MMM Do YY, h:mm:ss a'));
```
Comment (eric-ships) [inline src/index.js]: Do we need this here?

File: src/index.js [hunk 2]
```diff
@@ -129,6 +131,7 @@ async function check1080Ti() {
       : '1080Ti IS IN STOCK!!! 1080Ti IS IN STOCK!!! 1080Ti IS IN STOCK!!!';
 
     console.log(`${new Date().toTimeString()}: ${output}`);
+    console.log(moment().format('MMM Do YY, h:mm:ss a'));
```
Comment (eric-ships) [inline src/index.js]: .

File: src/index.js [hunk 3]
```diff
@@ -51,7 +52,7 @@ async function check1070() {
     const is1070OutOfStock = result.result.value.includes('Notify Me');
 
     if (!is1070OutOfStock) {
-      const body = '1070 IS NOW IN STOCK ' + NVIDIA_1070_URL;
+      const body = moment().format('MMM Do YY, h:mm:ss a') + ' 1070 IS NOW IN STOCK ' + NVIDIA_1070_URL;
```
Comment (eric-ships) [inline src/index.js]: Year seems unnecessary. Let's get rid of it!

**Review by eric-ships — COMMENTED** [review]
Nice! Just a couple of comments

**Review by txu89 — COMMENTED** [review]
Year removed from text.
Console.log combined and cleaned up.

**Review by eric-ships — APPROVED** [review]
Although formatting the console.log was not necessarily in scope, it's fine.

If we're going to format all the times and such, it might be nice to have a constant variable.

```javascript
const MOMENT_FORMAT = 'MMM Do, h:mm:ss a';
```

**Comment by txu89** [comment]
Added const for moment format

**Pull request merged**